tests/api/test_network.py

```diff
@@ -47,7 +47,8 @@ async def test_api_network_interface_update(aiohttp_client, coresys):
     """Test network manager api."""
     client = await aiohttp_client(await create_app(coresys))
```

Comment:
Let's make a fixture on `tests/conftest.py` for this client

Reply:
This will change for each part of the API that is tested.
Unless we add _all_ endpoints to the fixtrue

Reply:
Yes, let's do that

Reply:
That would be a good amount of duplicate code, I'll see if I find a smart way to do it